### PR TITLE
Remove the technical debt associated with using the model as a parame…

### PR DESCRIPTION
**Pull request by GeoffThorpeFT**
…ter to calls where the serializer is already present.

Serializer meta data already contains all model info
Added desc field to serializer meta to ensure consistent descriptive error messages

Many calls included a title, the model name and the list of attributes - all these fields are now redundant since the serializer meta contains all that info.

### CODE REVIEW

File: cmdbv2api/views.py [hunk 1]
```diff
@@ -120,8 +134,9 @@ def getList(request, model, serializer):
 
 
 # Update a record in a table
-def updateRecord(key, model, typeOfSerializer, dictFields):
+def updateRecord(key, typeOfSerializer, dictFields):
```
Comment (lucas42) [inline cmdbv2api/views.py]: What's the difference between serializer and typeOfSerializer?

File: cmdbv2api/views.py [hunk 2]
```diff
@@ -647,12 +663,11 @@ def historyByObject(request, object, **kwargs):
...
     """
     dictQuery = {'model': object}
-    attributes = ['action', 'model', 'datetime', 'originator', 'fields', 'fields__name', 'fields__value']
-    response = allRecords(request, ObjectHistory, ObjectHistorySerializer, attributes, dictQuery)
+    response = allRecords(request, ObjectHistorySerializer, dictQuery)
     return response
 
 @api_view(['GET'])
-def historyByOrigin(request, origininator, **kwargs):
+def historyByOrigin(request, originator, **kwargs):
```
Comment (lucas42) [inline cmdbv2api/views.py]: How did this function ever work?  And if it didn't, can we add a test for it?

File: cmdbv2api/views.py (hunk 1 above)
Comment (GeoffThorpeFT) [inline cmdbv2api/views.py]: nothing - i will make them the same; strange convention by previous developer

File: cmdbv2api/views.py (hunk 2 above)
Comment (GeoffThorpeFT) [inline cmdbv2api/views.py]: We have no bulk history tests - just tests of the history of individual items

Comment (lucas42) [inline cmdbv2api/views.py]: Might be worth adding regression tests as and when we find bugs.

**Comment by lucas42** [comment]
👍 This looks good.  Makes the codebase much more readable.

**Pull request merged**